.claude/hooks/post-edit-lint.sh

```
@@ -0,0 +1,93 @@
+
+# set -e は使わない。ツールの欠如やフォーマットエラーで
+# フック全体が中断するのを防ぎ、可能な限りフィードバックを返す。
+set -uo pipefail
+
+# CLAUDE_FILE_PATHS: 変更されたファイルパスのリスト (改行区切り)
+if [[ -z "${CLAUDE_FILE_PATHS:-}" ]]; then
+  exit 0
+fi
+
+# 先に cd してからパス解決する（相対パスが正しく解決されるように）
+cd "${CLAUDE_PROJECT_DIR:-.}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)